addons/compat_rhs_usf3/compat_rhs_usf3_csw/CfgVehicles.hpp

```diff
@@ -0,0 +1,80 @@
+class CfgVehicles {
+    class LandVehicle;
+    class StaticWeapon: LandVehicle {
+        class ACE_Actions;
+    };
+    class StaticMortar: StaticWeapon {
```

Comment:
```suggestion

    class StaticWeapon: LandVehicle {
        class ACE_Actions {
            class ACE_MainActions;
        };
    };
    class StaticMortar: StaticWeapon {};
```
I think this is what we want, it matches what we had before and how we do it in arf